### ISSUE TITLE
Importing curl code into Postman should detect and decode "x-www-form-urlencoded' data

### ISSUE THREAD

**Issue by veetil09** [a85/POSTMan-Chrome-Extension#3623]
## App Details:

```
Postman for Mac
Version 4.11.0
OS X 16.4.0 / x64
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there:

Encountered recently.

 2. Expected behaviour:

Sample ```cURL``` command generated by Postman:

```curl -X POST \
  https://dummyurl.mydomain.com/ \
  -H 'accept: application/json' \
  -H 'content-type: application/x-www-form-urlencoded' \
  -d 'config_type=global%3AmyOrg%3Aoptions'
```
Now if we try to import the above curl command, Postman does not decode the ``` config_type``` data correctly, like seen below:

<img width="597" alt="screen shot 2017-10-11 at 3 06 08 pm" src="https://user-images.githubusercontent.com/6054663/31460911-c9d9cda2-ae95-11e7-8ace-a280f5ca4582.png">

Expected result:

<img width="568" alt="screen shot 2017-10-11 at 2 46 48 pm" src="https://user-images.githubusercontent.com/6054663/31460931-d76df84e-ae95-11e7-8fb9-5fcdd575ce59.png">

Why should it be fixed?

Although the generated cURL code will work from the command line, but it can also be shared to import into Postman. In that case, Postman should be able to handle it.

**Comment by prashantagarwal** [comment]
@veetil09 Can you update to the latest version of Postman app and verify if it is still happening for you.

**Comment by euoia** [comment]
I am also experiencing this bug using Version 6.0.10 (6.0.10).

**Comment by MondoGao** [comment]
+1，can't decode a really long curl text copied from chrome

**Comment by yeoman88** [comment]
I am also experiencing this bug using Version v6.1.4

**Comment by veetil09** [comment]
@prashantagarwal Verry sorry to reply late. I can update and see if the issue still persists.

**Comment by abhijitkane** [comment]
Everyone - this has been fixed in Postman 7.2.1.
